task-bht_bold.psyexp

```diff
@@ -1031,16 +1031,16 @@
       <Param name="innerBorderColor" updates="None" val="black" valType="color"/>
       <Param name="innerBorderWidth" updates="None" val="2" valType="num"/>
       <Param name="innerFillColor" updates="None" val="green" valType="color"/>
-      <Param name="innerRadius" updates="None" val="0.0075" valType="num"/>
+      <Param name="innerRadius" updates="None" val="0.35" valType="num"/>
```

Comment:
I would need to see this on a 800x600 screen. Looks like the dot is going to be huge now (which is worse than too small)